docs/plugins/jenkins-pipeline.zh.md

```diff
@@ -64,8 +65,9 @@ tools:
       user: admin
       enableRestart: true
     scm:
-      projectURL: YOUR_JENKINS_ADDR/root/spring-demo
-      projectBranch: master
+      cloneURL: git@YOUR_REPO_CLONE_ADDRESS/root/spring-demo
```

Comment:
Please help to update `docs/best-practices/gitlab-jenkins-harbor.zh.md` as well.